### DISCUSSION TITLE
Wifi not saving.

### DISCUSSION THREAD

**Discussion by TheChief82** [rvdbreemen/OTGW-firmware#147] [Q&A] [answered]
Hello,

I used R vd Breemen his manual to install the firmware on my NodeMCU and after that to configure the Wifi. 
****************************************************************************************************
After first flash, or, when you alter your WiFi network, then you need to configure your wifi settings.

1. Wait for the ESP to go into the AP config mode.
2. Open your WiFi setting (smartphone or computer)
3. Connect to network OTGW-XX-XX-XX-XX-XX
4. A configuration screen should pop up. If this does not happen, just go to 192.168.4.1
5. Now configure your WiFi setup.
6. After the device is connected to WiFi, go to http://otgw.local/ or enter the IP address of your ESP
****************************************************************************************************

After installing the firmware it reboots and the two blue leds keep burning constantly, is that normal?
I can find and connect to the AP (OTGW-XX-XX-XX-XX-XX) and get the WifiManager screen. 
When i configure the WiFi by selecting the correct SSID and entering the correct pasword and hitting save it says:
**************************************
Saving Credentials
Trying to connect ESP to network.
If it fails reconnect to AP to try again
**************************************
After this message pops up nothing happening, when i reconnect to my normal wifi connection i also keep seeing the
AP of the OTGW. When i open up the router page i can see the MAC adress of the NodeMCU is connected to the router but when
entering its IP adress in the internetbrowser it does not open a website/portal.

Any help would be welcome!!

**Comment by TheChief82** [comment]
When reconnecting to the AP it says at the bottom of the WiFiManager screen:
"Not connected to (my wifi SSID)". 

Reconfiguring the wifi or to another network gives me the same problem

**Comment by rvdbreemen** [comment]
Hi Chief

It sounds to me that the password does not take. Could you try another password just to see if that's not the issue. 

Normally it should connect instantly after configuration. 

Robert

**Comment by rvdbreemen** [comment, marked as answer]
Did you get this solved, if not try soon to be released version.

**Reply by TheChief82** [reply]
Helllo Robert,

I missed you're reply from last the 20th of November 2022. 
Yes i solved the problem, don't know anymore what i did exactly....

**Comment by wibbit** [comment]
Morning all, was this ever resolved?

I am seeing a similar issue, however I do not believe it's related to credentials.

I have to AP's in my house, an Asus RT-AC3200 and an RT-AX86U.

There are no issues connecting to the RT-AC3200 AP, however the connectivity back to the RT-AX86U is incredibly unreliable.
It will connect briefly and then disconnect, I'm seeing ~90% packet drops, however when there is connectivity it appears to be clustered together.

I found https://github.com/esp8266/Arduino/issues/8299 which seems to be similar, and have tried assigning a reserved IP with no luck.

I've also attempted to add WiFi.setPhyMode(WIFI_PHY_MODE_11G) that I don't believe has made a huge difference.

This weekend I may try and work out how to statically assign an IP on the device it self, and see if that improves things at all.

I'm very new to this, and have no idea how to debug on the device itself, will have to look into that.

**Comment by Basb1234** [comment]
I have the same issue as reported by 'TheChief82'. I am not able to connect to a wifi network if my Asus RT-AX88U is supporting dual-band (2.4 Ghz and 5 Ghz) with 'smart-connect'. The 'smart-connect' functionality shows 1 wifi network (Same SSID and password) for devices, but depending on the connection quality (RSSI) it connect a device to either the 5 Ghz or 2.4 Ghz network. 

I can connect to the OTFW if the smart-connect is turned off. It that case I have a 2.4 Ghz and a seperate 5 Ghz netowrk . I would like to use 'smart-connect' functionality together with OTGW. 

I have 2 questions:
1 - Do you have an idea why OTGW doesn't work in combination with smart connect?
2 - The 'smart-connect' can be configured (see below). Do you have any suggestion which settings could make it work? Or other router setting?

The 2.4 Ghz and 5 Ghz band settings:
![image](https://github.com/user-attachments/assets/26c5e5a0-41ac-45e8-ab50-d36d92765a3e)


Below the smart connect settings:
![image](https://github.com/user-attachments/assets/c930e45d-0fc9-40d3-8489-1417841047ee)

Help would be appreciated.
